### PR TITLE
Migrate existing HAB PKI terraform config to terragrunt

### PR DESCRIPTION
**Pull request by AlCutter**
This creates a terragrunt `ci` env which is directly mapped to the existing CI HAB PKI instantiation.

```bash
~/armored-witness/deployment/hab_pki/live/ci$ terragrunt plan
...
Plan: 8 to add, 0 to change, 8 to destroy.
```
The 8 mods are expected and due to certificate lifetimes being truncated due to the passage of time.

### CODE REVIEW

File: deployment/hab_pki/modules/hab_pki/main.tf [hunk 1]
```diff
@@ -33,7 +33,7 @@ resource "google_project_service" "privateca_api" {
 # KMS key rings & data sources
 resource "google_kms_key_ring" "hab" {
   location = var.signing_keyring_location
-  name     = "hab-ci"
+  name     = format("hab-%s", var.env)
```
Comment (mhutchinson) [inline deployment/hab_pki/modules/hab_pki/main.tf:36]:
```suggestion
  name     = "hab-${var.env}"
```

Does this work? It's the style I used throughout the other configs, and that you're already using in the hcl files.

**Review by mhutchinson — COMMENTED** [review]
Looks ace!

Only big comment is about what the state of the state is? (Just Dropped In). By which I mean, the terragrunt defines the gcs bucket now, but it's still defined quite throughly in the main module file from lines 310 onwards.

File: deployment/hab_pki/modules/hab_pki/main.tf (hunk 1 above)
Comment (AlCutter) [reply deployment/hab_pki/modules/hab_pki/main.tf:36]: bah.

Comment (mhutchinson) [reply deployment/hab_pki/modules/hab_pki/main.tf]: 🐑

**Comment by AlCutter** [comment]
> Looks ace!
> 
> Only big comment is about what the state of the state is? (Just Dropped In). By which I mean, the terragrunt defines the gcs bucket now, but it's still defined quite throughly in the main module file from lines 310 onwards.

done - ta!

**Review by mhutchinson — APPROVED** [review]

**Pull request merged**